deployment/run_server

```diff
@@ -1,10 +1,34 @@
 #!/bin/bash -xe
 
 fc-cache -v /usr/share/fonts/user
+set -e
+
+BUILD_DIR=/tmp/plotly-graph-exporter-build
```

Comment:
I'm curious, why start from `/tmp/` ? This repo has already a `build/` directory, perhaps that would be cleaner?